wye/profiles/models.py

```diff
@@ -29,17 +30,27 @@ class Meta:
         ordering = ('-id',)
 
     def __str__(self):
-        return '{} - {}'.format(self.slug, self.display_name)
+        return '{}'.format(self.display_name)
 
 
 class Profile(models.Model):
     user = models.OneToOneField(User, primary_key=True, related_name='profile')
-    # the slug fields become the username and should be unique for each user
-    slug = models.CharField(max_length=100, unique=True)
+    slug = models.SlugField()
     mobile = models.CharField(max_length=10)
     usertype = models.ManyToManyField(UserType)
     interested_sections = models.ManyToManyField(WorkshopSections)
     interested_locations = models.ManyToManyField(Location)
+    location = models.ForeignKey(Location, related_name="user_location", null=True)
+    github = models.URLField(null=True, blank=True)
+    bitbucket = models.URLField(null=True, blank=True)
+    facebook = models.URLField(null=True, blank=True)
+    googleplus = models.URLField(null=True, blank=True)
+    linkedin = models.URLField(null=True, blank=True)
+    twitter = models.URLField(null=True, blank=True)
+    stackoverflow = models.URLField(null=True, blank=True)
+    slideshare = models.URLField(null=True, blank=True)
+    reddit = models.URLField(null=True, blank=True)
```

Comment:
@kracekumar  @theskumar 

How about keeping Following? (Because, they will help tutor to describe his/her skills
)
* Github
* Bitbucket
* Linkedin
* Stackoverflow
* Slideshare

And removing following? (Because it will help less in describing tutor skills):
* Twitter
* Facebook
* Googleplus
* Reddit